### DISCUSSION TITLE
Welcome to violentmonkey Discussions!

### DISCUSSION THREAD

**Discussion by gera2ld** [violentmonkey/violentmonkey#1123] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JetLua** [comment]
![DonutTheDog-AgADUQADrWW8FA](https://user-images.githubusercontent.com/6738986/101570173-d1485b80-3a10-11eb-84cd-e13528542652.gif)
吧主🐂🍺
